tools/webots.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import shutil
+import subprocess
+import sys
+import textwrap
+from pathlib import Path
+
+from termcolor import cprint
+
+import b
+
+
+def register(command):
+    command.help = "Build and run images for use with Webots"
+    subparsers = command.add_subparsers(help="sub-command help", dest="sub_command")
+
+    build_subcommand = subparsers.add_parser("build", help="Build the docker image for use with Webots")
+    build_subcommand.add_argument("roles", nargs="+", help="The roles to build for the image")
+
+    push_subcommand = subparsers.add_parser(
+        "push",
+        help=textwrap.dedent(
+            """
+            Push the simulation docker image to the TC registry. Run the aws CLI and login with the TC credentials before running this command.
+            See "Uploading Docker Images" in the Robocup "API Specifications for Virtual Soccer Competition" document for details.
```

Comment:
Is there are way to check that we are logged in properly? If you can detect this it would be nice to get a clean exit if login hasnt happened

Reply:
Docker will show an authentication error if we attempt to push without first logging in.